src/finalizer/finalizer.nativeproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.DotNet.CMake.Sdk">
+
+  <PropertyGroup>
+    <!-- We use forward slashes in the ArtifactsDir path because backslashes in the CMakeLists.txt are escape characters. -->
+    <ArtifactsDirForCMake>$([System.String]::Copy($(ArtifactsDir)).Replace('\','/'))</ArtifactsDirForCMake>
+    <CMakeLists>CMakeLists.txt</CMakeLists>
+  </PropertyGroup>
+
+  <!-- This will be passed through -D and referenced by the CMakeLists.txt to correctly resolve the platform specific libs. -->
+  <ItemGroup>
+    <CMakeDefines Include="Platform" Value="$(Platform)" />
+    <CMakeDefines Include="ArtifactsDir" Value="$(ArtifactsDirForCMake)" />
+    <!-- Include this item below to create a binlog for the native build. -->
+    <!-- <CMakeNativeToolArguments Include="/bl:$(ArtifactsDirForCMake)native.binlog" /> -->
```

Comment:
Any reason why this is commented out? Do we want it enabled by default or otherwise hide it behind a conditional property? Having binlogs for everything is useful when investigating build failures I think

Reply:
It was there in the file already (meaning, it was commented out in the file) and I figured that you put it there for yourself. Lol. I didn't look at the blame on it, though. I haven't actually tested with it being on, but I can do that and see if we just want to keep it on always.